datafusion/physical-expr/src/planner.rs

```diff
@@ -286,19 +286,13 @@ pub fn create_physical_expr(
 
                 let list_exprs = list
                     .iter()
-                    .map(|expr| match expr {
-                        Expr::Literal(ScalarValue::Utf8(None)) => create_physical_expr(
```

Comment:
both match arms are the same. It is interesting clippy didn't tell us about that